### PR TITLE
[feat] Add ability to compare the selected runs from the table

### PR DESCRIPTION
**Pull request by arsengit**
- Added the ability to inspect selected runs from the table 
- Set the `MenuItem` size small by default in the `ControlPopover` component

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #1313

File: aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        window.open(`/${value}?select=${search}`, '_blank');
+        window.focus();
+      }
+    },
+    [appName, selectedRows],
+  );
+
+  return (
+    <ErrorBoundary>
+      <ErrorBoundary>
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx:58]: remove `ErrorBoundary ` duplication

File: aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+}: ICompareSelectedRunsProps): React.FunctionComponentElement<React.ReactNode> {
+  const onCompare: (value: string) => void = React.useCallback(
+    (value: string) => {
+      if (value) {
+        const runHashArray: string[] = _.uniq([
+          ...Object.values(selectedRows).map((row: any) => row.runHash),
+        ]);
+
+        const query = `run.hash in [${runHashArray
+          .map((hash) => {
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx:34]: let's simplify the body of map

File: aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          component={
+            <div className='CompareSelectedRunsPopover'>
+              {EXPLORE_SELECTED_RUNS_CONFIG[appName as AppNameEnum].map(
+                (item: AppNameEnum) => (
+                  <MenuItem
+                    className='CompareSelectedRunsPopover__item'
+                    onClick={() => onCompare(item)}
+                    key={item}
+                  >
+                    <Icon box name={item as any} />
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.tsx:93]: let's replace as `any` to `string`

File: aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+@use 'src/styles/abstracts' as *;
+
+.CompareSelectedRunsPopover {
+  padding: 0.5rem;
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover.scss:4]: let's use space variable here

File: aim/web/ui/src/components/Table/Table.tsx [hunk 5]
```diff
@@ -23,6 +23,7 @@ import SortPopover from 'pages/Metrics/components/Table/SortPopover/SortPopover'
 import ManageColumnsPopover from 'pages/Metrics/components/Table/ManageColumnsPopover/ManageColumnsPopover';
 import HideRowsPopover from 'pages/Metrics/components/Table/HideRowsPopover/HideRowsPopover';
 import RowHeightPopover from 'pages/Metrics/components/Table/RowHeightPopover/RowHeightPopover';
+import CompareSelectedRunsPopover from 'pages/Metrics/components/Table/CompareSelectedRunsPopover/CompareSelectedRunsPopover';
```
Comment (KaroMourad) [inline aim/web/ui/src/components/Table/Table.tsx:26]: let's shorten the import

**Review by KaroMourad — APPROVED** [review]

**Review by roubkar — APPROVED** [review]

**Pull request merged**